src/Firebase.Database/Database.uno

```diff
@@ -656,6 +656,14 @@ namespace Firebase.Database
         }
     }
 
+    internal class ReadByQueryEqualToValue : Promise<string>
```

Comment:
You have to declare this class as `extern(!mobile)` We are getting an error in Mobile build with a duplicate declaration.